tests/expectations/compiler/const_generics/basic.out

```diff
@@ -1,5 +1,4 @@
 program test.aleo;
-
```

Comment:
To avoid this massive diff which causes github to slow down significantly, maybe make sure these files don't need to be changed? You can change that in another separate (trivial) PR if you like.